sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/http/policy/RetryPolicy.java

```diff
@@ -76,7 +76,7 @@ public RetryPolicy() {
      * @param retryOptions The {@link RetryOptions} used to configure this {@link RetryPolicy}.
      * @throws NullPointerException If {@code retryOptions} is null.
      */
-    RetryPolicy(RetryOptions retryOptions) {
+    public RetryPolicy(RetryOptions retryOptions) {
```

Comment:
is it intentional?

Reply:
yeah, didnt mean to pull it into this PR. But was a result of a quick discussion when getting parity for `public HttpLoggingPolicy(HttpLogOptions httpLogOptions)`.